python_course_1604/class_01/tests/test_04.py

```diff
@@ -12,6 +12,7 @@
 
 
 @skip_if_exercize_not_started(exercize)
+@unittest.skip('It brokes coverage.')
```

Comment:
I would prefer this step would skip only on your computer to have it launched on CI server.

Please change this with following:
```Python
@unittest. skipIf(unittest.skipIf(socket.gethostname() == 'your-host-name')))
```
Where _your_host_name_ is your full host name.